### DISCUSSION TITLE
sdrplay

### DISCUSSION THREAD

**Discussion by IK8HCG** [AlexandreRouma/SDRPlusPlus#1065] [General]
I deleted the SDRUNO software from my PC, now I can no longer start SDR++, SDRplay no longer appears in the Source-Audio list. Do I have to keep SDRUNO in HD?
tnx

**Comment by AlexandreRouma** [comment]
You need to have the SDRplay API (aka the sdrplay driver) installed for SDR++ to work with SDRplay devices. Uninstalling SDRuno likely uninstalled this API too. You can download the API standalone from SDRplay's website.

**Comment by IK8HCG** [comment]
in which SDR++ folder must be downloaded

**Comment by starvald** [comment]
Its an exe file just download then install.
